text/069-coq-roadmap.md

```diff
@@ -0,0 +1,444 @@
+requiring changes in the Coq system, it includes availability of persons to
+review and merge the changes.
+
+# Motivation
+
+Producing a roadmap for the Coq project is important for several reasons:
+
+- It helps the community to know what to expect from the Coq project in the
+  future, in particular what changes can reasonably be expected to become
+  part of next releases, and what changes are more likely to be delayed
+  because of lack of resources.
+
+- It helps Coq developers to choose priorities together, and to work more
+  efficiently toward those by making sure that their work will be supported
+  by other developers, including reviewers.
+
+- It helps contributors to know in what areas their contribution is likely
+  to receive the most support from the Coq developers.
+
+- It helps highlighting other priorities where progress is not possible for
+  lack of resources, and where the Coq project should aim to find
+  more resources.
+
+# Detailed design
+
+The Coq roadmap is declined at three levels: short, medium and long term.
+Each of these levels address different needs, but they should be checked
+for consistency with each other.
+
+- The short term roadmap is focused on priorities on which we are committed
+  to progress in the coming months and that are thus the most likely to
+  actually happen. We define a commitment from the Coq team as a commitment
+  of several (at least two) contributors, including at least one relevant
+  maintainer to review the work proposed by other contributors. The work
+  can include implementation work when the design is already clear, work on
+  the design from ideas that are well understood, or even exploratory work,
+  but we should only list the parts that are likely to converge in the
+  specified time-frame (i.e., if it is not clear if exploratory work will
+  lead to a solution that can be implemented, we should only commit on the
+  exploratory work).
+
+- The medium term roadmap is focused on areas where we see important progress
+  being needed, which could take several years to complete, and for which we
+  do not always have the ressources needed. It is used to guide the renewal
+  of the short term roadmap, but also the search for new resources to make
+  progress in these areas. The medium term roadmap should contains high level
+  descriptions of the areas requiring progress, but can contain rough plans
+  for how to proceed to make progress in these areas if the team already has a
+  vision about the steps needed.
+
+- The long term roadmap is focused on what our vision for the future of Coq is
+  and not necessarily how we are going to achieve it. It is used to provide
+  general ideas that we can check our medium and short term roadmaps against,
+  and to give users more clarity on what the future of Coq could look like. It
+  should include our vision of what kind of users Coq will target in the future.
```

Comment:
I think these thoughts are out of place.  We should eventually state the goals for Coq and describe some of the principal user groups.  Then items in the roadmap fit into those goals.  That should earlier in the document, maybe at the beginning of the motivation section.  But maybe not fleshed out in any detail in this version of the roadmap.